packages/wallets/ledger/src/polyfills/Buffer.ts

```diff
@@ -0,0 +1,7 @@
+import { Buffer } from 'buffer';
+
+if (window.Buffer === undefined) {
```

Comment:
Hmm do we need to check if `typeof window !== 'undefined'` first?

Reply:
I could do that! I think I just figured that if someone is using a wallet adapter in a non-browser context, then they're gonna have a bad time anyway, but you're probably right. I could see this breaking in SSR or something.